receptor_affinity/mesh.py

```diff
@@ -0,0 +1,459 @@
+import atexit
```

Comment:
I renamed this module from `affinity` to `mesh`. Please let me know if you have any other name suggestion